Modules/CaPTkSurvival/include/CaPTkFeatureExtractionClass.h

```diff
@@ -0,0 +1,105 @@
+/**
+\file  FeatureExtractionClass.h
+
+\brief Declaration of the FeatureExtractionClass
+
+https://www.med.upenn.edu/sbia/software/ <br>
+software@cbica.upenn.edu
+
+Copyright (c) 2018 University of Pennsylvania. All rights reserved. <br>
+See COPYING file or https://www.med.upenn.edu/sbia/software-agreement.html
+
+*/
+#ifndef CaPTkFeatureExtractionClass_h
+#define CaPTkFeatureExtractionClass_h
+
+// The following header file is generated by CMake and thus it's located in
+// the build directory. It provides an export macro for classes and functions
+// that you want to be part of the public interface of your module.
+#include <MitkCaPTkSurvivalExports.h>
+
+#include "itkVariableSizeMatrix.h"
+
+#include <vector>
+
+namespace captk 
+{
+
+class MITKCAPTKSURVIVAL_EXPORT FeatureExtractionClass
+{
+public:
+  //!Constructor
+  FeatureExtractionClass();
+  
+  //!Destructor
+  ~FeatureExtractionClass()
+  {
+    // do nothing
+  }
+
+  const int TRAINING_LABEL_NEAR = 0;
+  const int TRAINING_LABEL_FAR  = 1;
+
+  using VariableSizeMatrixType = itk::VariableSizeMatrix<double>;
+  using VectorDouble = std::vector<double>;
+
+  //!Returns the training data
+  VariableSizeMatrixType
+  GetTrainingData()
+  {
+    return mTrainingData;
+  }
+
+  //!Sets the value of training data
+  void SetTrainingData(VariableSizeMatrixType &inputTrainingData)
+  {
+    mTrainingData = inputTrainingData;
+  }
+
+  //!Returns the test data
+  VariableSizeMatrixType GetTestData()
+  {
+    return mTestData;
+  }
+
+  //!Sets the value of test data
+  void SetTestData(VariableSizeMatrixType &inputTestData)
+  {
+    mTestData = inputTestData;
+  }
```

Comment:
Implementations should be in cpp file (in src/)

Reply:
Done, will update this PR momentarily